### DISCUSSION TITLE
Where can I find retranslators list?

### DISCUSSION THREAD

**Discussion by 3DRaven** [mastodon/mastodon#21043] [General]
Hi! I created mastodon server 
https://mtdn.anyqn.com
Just for testing. How can I get messages from fediverse? I found in settings possibility to set "retranslator" but not found any public retranslators. Where can I find list of retranslators? Without feed of random messages server is boring :(

**Comment by mjankowski** [comment]
If this is still relevant, you can try -- https://relaylist.com
